src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx

```diff
@@ -199,139 +228,117 @@ Global decisions cover a variety of correlation scenarios. Review descriptions t
+  alt="A screenshot of the decision builder showing how to name a decision."
+  src={decisionBuilderName}
+/>
 
-    <tr>
-      <td>
-        Step 3: Correlate topology (optional)
-      </td>
+### Step 5: Use advanced settings [#advanced-settings]
 
-      <td>
-        For automatic topology correlation, make sure your telemetry data is collected by [New Relic agents](/docs/new-relic-solutions/new-relic-one/install-configure/compatibility-requirements-new-relic-agents-products/). Learn more about [topology correlation out-of-the-box](#topology-requirements).
+Use the advanced settings area to further customize how your decision behaves when correlating events. Each setting has a default value so customization is optional.
 
-        In addition, you can setup topology data via our NerdGraph `aiTopologyCollector`. (Search for `aiTopology` in the [NerdGraph GraphiQL explorer](https://api.newrelic.com/graphiql)). This allows any topology-related decision to be matched with your topology data. Learn more about [setting up topology correlation](#topology).
-      </td>
-    </tr>
+* **Time window**: Sets the maximum time between two incidents created time for them to be eligible for correlation.
+* **Issue priority**: Overrides the default priority setting (`inherit priority`) to add higher or lower priority if the incidents are correlated.
+* **Frequency**: Modifies the minimum number of incidents that need to meet the rule logic for the decision to trigger.
+* **Similarity**: If you're using `similar to` operators in your rule logic, you can choose from a list of algorithms and set its sensitivity. This will apply to all `similar to` operators in your decision.
 
-    <tr>
-      <td>
-        Step 4: Give it a name
-      </td>
+<img
+  title="Decision - advanced settings"
+  alt="A screenshot of the decision builder showing how to configure advanced settings."
+  src={decisionBuilderSettings}
+/>
 
-      <td>
-        After you configure your decision logic, give it a recognizable name and description. This is used in notifications and other areas of the UI to indicate which decision caused a pair of incidents to be correlated together.
-      </td>
-    </tr>
+## Logic operators [#operators]
 
-    <tr>
-      <td>
-        Step 5: Use advanced settings (optional)
-      </td>
+Decision provides a set of operators to help you flexibly define how an incident's attribute value evaluates in a logic filter.
+The basic ones are **equals**, **contains**, **starts with**, **ends with**, **exists**, and their negate operators accordingly. For example, **does not equal**.
 
-      <td>
-        Optional. Use the advanced settings area to further customize how your decision behaves when correlating events. Each setting has a default value so customization is optional.
+There is a similarity operator **is similar to**, the underlying [similarity algorithm](#algorithms) can be specified for this operator. By default, it uses Levenshtein Distance.
```

Comment:
this sentence needs some work; it's two sentences run together seems like.  

what is 'levenshtein distance'? I assume it's some sort of algorithm, but might want to specify that's what it is.

Reply:
thanks.